src/main/scala/no/ndla/imageapi/model/api/Error.scala

```diff
@@ -18,7 +18,7 @@ import scala.annotation.meta.field
 @ApiModel(description = "Information about errors")
 case class Error(@(ApiModelProperty@field)(description = "Code stating the type of error") code: String,
                  @(ApiModelProperty@field)(description = "Description of the error") description: String,
-                 @(ApiModelProperty@field)(description = "When the error occured") occuredAt: String = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss.SSS").format(new Date()))
+                 @(ApiModelProperty@field)(description = "When the error occurred") occurredAt: String = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss.SSS").format(new Date()))
```

Comment:
Hmm. Dette er jo strengt tatt en API-endring... Hvordan ville det truffet dere på "konsument-siden", @runarfu ?

Reply:
Ah, det er sant! Vi fikset dette lokalt hos oss, siden vi ikke er i prod ennå og API-endringer er derfor greit. Jeg kan godt bare fikse dette i vår fork, så trenger ikke dere å forholde dere til det. Det er jo en uviktig ting.

Reply:
Mm. Men egentlig er jeg jo for å fikse, ellers blir det bare liggende. 

Vi lar pull-requesten ligge. For oss fungerer det også greit å bare ta det inn, men må sjekke med vår eneste andre eksterne bruker...